flask_rebar/rebar.py

```diff
@@ -357,6 +363,17 @@ def set_default_authenticators(self, authenticators):
        """
         self.default_authenticators = authenticators or []
 
+    def load_handlers(self, recursive=False):
+        """
+        Load modules in ``self.handlers`` package
+        :param boolean recursive:
+            recursively look through packages.
+            Default: False
+        """
+        for package in self.handlers:
+            for handler in find_modules(package, recursive=recursive):
+                importlib.import_module(handler)
```

Comment:
Should there be any error handling here or are we ok with just bailing out if we don't get the right input?